conjure-java-core/src/integrationInput/java/com/palantir/product/EnumExample.java

```diff
@@ -2,7 +2,9 @@
 
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonValue;
+import com.google.common.collect.ImmutableList;
```

Comment:
I think we avoid using guava in some configurations, so it's safer to use `Collections.unmodifiableList(Arrays.asList(<values>))`

Reply:
Ah, I checked and saw there was a dependency on guava, so preferred immutable over unmodifiable. I've switched now.